modules/android/pages/releasenotes.adoc

```diff
@@ -10,3 +10,5 @@ include::partial$_set_page_context_for_android.adoc[]
 
 [#maint-latest]
 include::partial$release-notes/couchbase-mobile-android-release-note.3.2.0.adoc[]
+
+include::partial$release-notes/couchbase-mobile-android-release-note.3.2.0-beta.adoc[]
```

Comment:
why did we decide to include the -beta relnotes again?

Reply:
They were public releases so it was decided to retain them for visibility. PM ask